app/lib/bk/compat/parsers/bitrise/env_helper.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    class Bitrise
+      # Helper methods for processing environment variables in Bitrise configuration
+      module EnvHelper
+        def self.process_env_variable(env, target_env)
+          return unless env.is_a?(Hash)
+
+          env.each do |key, value|
+            next if key == 'opts' || key.to_s.strip.empty? || value.to_s.strip.empty?
+
+            target_env[key] = value
+          end
```

Comment:
an easier way to do this is:

```suggestion
          env.delete!('opts')
          env
```